src/middlewared/middlewared/plugins/idmap.py

```diff
@@ -787,7 +847,13 @@ class Config:
     )
     async def do_create(self, data):
         """
-        Create an entry in the idmap backend table.
+        `domain` specifies the domain for which the idmap backend is being created. Numeric id, short-form
```

Comment:
This repeated identical comment should be factored out into a single comment at the start of the section, then referred back to from these multiple entries.